auth-plugin.c

```diff
@@ -18,6 +18,11 @@
 
 #include "go-auth.h"
 
+// Same constant as one in go-auth.go.
+#define AuthRejected 0
```

Comment:
Wait, this seems odd, weren't these constants defined already in your previous PR?